test/sql.test.js

```diff
@@ -247,12 +247,18 @@ describe('sql connector', function() {
     });
   });
 
-  it('builds where and ignores invalid clauses in or', function() {
-    const where = connector.buildWhere('customer', {
-      name: 'icecream',
-      or: [{notAColumnName: ''}, {notAColumnNameEither: ''}],
-    });
-    expect(where.sql).to.not.match(/ AND $/);
+  it('builds where and throws if invalid clauses in or', function() {
+    let errorThrown = false;
+    try {
+      connector.buildWhere('customer', {
+        name: 'icecream',
+        or: [{notAColumnName: ''}, {notAColumnNameEither: ''}],
+      });
+    } catch (e) {
+      errorThrown = true;
+      expect(e.message).to.eql('Unknown property notAColumnName used for model customer');
+    }
+    expect(errorThrown).to.eql(true);
```

Comment:
Please use [`throw`](https://www.chaijs.com/api/bdd/#method_throw) assertion.

```js
expect(
  () => connector.buildWhere('customer', /*...*/)
).to.throw(
  'Unknown property notAColumnName used for model customer'
);
```

The same comment applies to other newly added tests too.

Reply:
Done